### DISCUSSION TITLE
Footnotes from HTML are not rendered natively as footnotes

### DISCUSSION THREAD

**Discussion by mbanerjeepalmer** [jgm/pandoc#10438] [Q&A]
Using the following idiom (taken from https://html.spec.whatwg.org/multipage/semantics-other.html#footnotes)
```
    <p> Announcer: Number 16: The <i>hand</i>.
    <p> Interviewer: Good evening. I have with me in the studio tonight
        Mr Norman St John Polevaulter, who for the past few years has been
        contradicting people. Mr Polevaulter, why <em>do</em> you
        contradict people?
    <p> Norman: I don't. <sup><a href="#fn1" id="r1">[1]</a></sup>
    <p> Interviewer: You told me you did!</p>

    <section>
        <p id="fn1"><a href="#r1">[1]</a> This is, naturally, a lie,
            but paradoxically if it were true he could not say so without
            contradicting the interviewer and thus making it false.</p>
    </section>
```

Using commonmark as the output:
```
Announcer: Number 16: The *hand*.

Interviewer: Good evening. I have with me in the studio tonight Mr Norman St John Polevaulter, who for the past few years has been contradicting people. Mr Polevaulter, why *do* you contradict people?

Norman: I don't. <sup><a href="#fn1" id="r1">[1]</a></sup>

Interviewer: You told me you did!

<div class="section">

[\[1\]](#r1) This is, naturally, a lie, but paradoxically if it were true he could not say so without contradicting the interviewer and thus making it false.

</div>
```

Instead what is expected is native footnotes: https://pandoc.org/chunkedhtml-demo/8.19-footnotes.html

I can't see any [other questions](https://github.com/jgm/pandoc/discussions?discussions_q=html+footnotes) about this. I must be  doing something wrong at a basic level.

There is #3724 , but that focuses on reversibility, not getting it right in one shot.

Thanks!

**Comment by jgm** [comment]
See #5294.  In your case you don't have the aria-roles, so #5294 wouldn't help -- but I think that using the aria-roles is the only really reliable way to distinguish footnotes from other sorts of links.

**Reply by mbanerjeepalmer** [reply]
Thanks for clarifying. Is it worth mentioning in the docs that this isn't supported at the moment?

As for implementation, agree that aria would be a good start. My uneducated opinion is that the WHATWG idiom can safely said to be a footnote. What else would a superscript within an anchor (or vice versa) which references another anchor tag that references the original, be? I have no idea what the implementation implications are, though. I may throw together a custom lua script using these principles and see how that works.
